l10n_ch_hr_payroll/models/hr_contract.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # © 2012 David Coninckx (Opennet Sarl)
+# Make additional fields translatable by Ermin Trevisan (twanda AG)

```

Comment:
Only copyright here. The explanation should be in the commit message or the global PR